python/apps/taiga/src/taiga/comments/repositories.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Copyright (c) 2023-present Kaleidos INC
+
+from typing import Literal, TypedDict
+from uuid import UUID
+
+from asgiref.sync import sync_to_async
+from taiga.base.db.models import QuerySet
+from taiga.comments.models import Comment
+
+##########################################################
+# filters and querysets
+##########################################################
+
+
+DEFAULT_QUERYSET = Comment.objects.all()
+
+
+class CommentsFilters(TypedDict, total=False):
+    story_id: UUID
```

Comment:
don't use `story`, story is not a generic name

Reply:
Right, changed to object_id